src/component-generators/vue/vue-component.ts

```diff
@@ -44,22 +46,23 @@ const createVueGenerator = (
     const cssChunks = result.chunks.filter((chunk) => chunk.meta.fileId === 'vuecss')
     const htmlChunks = result.chunks.filter((chunk) => chunk.meta.fileId === 'vuehtml')
 
-    const jsCode = chunksLinker.link(jsChunks)
-    const cssCode = chunksLinker.link(cssChunks)
-    const htmlCode = chunksLinker.link(htmlChunks)
+    const jsCode = removeLastEmptyLine(chunksLinker.link(jsChunks))
+    const cssCode = removeLastEmptyLine(chunksLinker.link(cssChunks))
+    const htmlCode = removeLastEmptyLine(chunksLinker.link(htmlChunks))
 
     return {
-      code: `
-<template>
-${htmlCode}</template>
+      code: `<template>
+${addSpacesToEachLine(2, htmlCode)}
+</template>
 
 <script>
-${jsCode}</script>
+${addSpacesToEachLine(2, jsCode)}
```

Comment:
as far as I know, Vue developers prefer to not have the script and styles indented, only the template.

A few examples (didn't find one with both styles and script):
https://vuejs.org/v2/style-guide/#Element-selectors-with-scoped-use-with-caution
https://github.com/gothinkster/vue-realworld-example-app/blob/master/src/components/TagList.vue